mlflow/store/abstract_store.py

```diff
@@ -45,19 +45,24 @@ def get_experiment(self, experiment_id):
         Fetches the experiment by ID from the backend store.
 
         :param experiment_id: Integer id for the experiment
-        :return: A single Experiment object if it exists, otherwise None.
+        :return: A single :py:class:`mlflow.entities.Experiment` object if it exists,
+        otherwise raises an exception.
```

Comment:
nit: Could we indent newlines like https://github.com/kennethreitz/requests/blob/master/requests/api.py?